package.json

```diff
@@ -22,7 +22,6 @@
     "bundlesizecollect": "node ./scripts/bundle-size-collect",
     "change": "beachball change --scope \"!packages/fluentui/*\"",
     "check-for-modified-files": "cd scripts && just-scripts check-for-modified-files",
-    "checkchange": "beachball check  --scope \"!packages/fluentui/*\" --changehint \"Run 'yarn change' to generate a change file\"",
```

Comment:
I'd keep this, so folks can still check this on local machine, just add `npx` so on local/ADO it will invoke installed beachball and in  Github Action it will install temporarily. Only con is that we loose the specific version 😶‍🌫️

Reply:
Makes sense, restored it.

Reply:
Losing the specific version in this case could be a problem. Could we add a step to the workflow to install the version of beachball specified in package.json?